client/src/components/QandA/AddAnswer.jsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+class AddAnswer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      answerEntry: '',
+      nicknameEntry: '',
+      emailEntry: '',
+      addedPhotos: []
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(e) { // this function isn't yet implemented. FIX ME
+    if (e.target === 'answer') { // determine distinguishing target value for corresponding input
+      this.setState({ answerEntry: e.target.value });
+    }
+    if (e.target === 'nickname') {
+      this.setState({ nicknameEntry: e.target.value });
+    }
+    if (e.target === 'email') {
+      this.setState({ emailEntry: e.target.value });
+    }
+  }
+
+  handleClick() {
+    // separate modal allowing up to five photos to be submitted. Should show thumbnails of each added image.
+    // after five photos uploaded, button disappears.
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    // ensure that both answer and email fields have been filled out and are valid formats via helper functions
```

Comment:
I think you could add the required='true' attribute in the HTML tag to require the field. 
And you might be able to handle the [email vallidation](https://stackoverflow.com/questions/5601647/html5-email-input-pattern-attribute) in the HTML tag too